### ISSUE TITLE
Payara 5 alpha 3 https is working only on safari

### ISSUE THREAD

**Issue by ionutursuleanu** [payara/Payara#2144]
# Description #
----------

Payara 5 alpha 3 https is working only on safari. The https://localhost:8181 displays 

> This site can’t be reached
> 
> localhost unexpectedly closed the connection.
> Try:
> Checking the connection
> Checking the proxy and the firewall
> Running Network Diagnostics
> ERR_CONNECTION_CLOSED

Server log

```
[2017-11-10T20:17:07.600+0200] [Payara 5.0.Alpha3] [WARNING] [] [org.glassfish.grizzly.filterchain.DefaultFilterChain] [tid: _ThreadID=39 _ThreadName=http-thread-pool::http-listener-2(4)] [timeMillis: 1510337827600] [levelValue: 900] [[
  GRIZZLY0013: Exception during FilterChain execution
java.lang.NoSuchMethodError: sun.security.ssl.SupportedEllipticCurvesExtension.curveIds()[I
	at sun.security.ssl.ServerHandshaker.setupEphemeralECDHKeys(ServerHandshaker.java:1499)
	at sun.security.ssl.ServerHandshaker.trySetCipherSuite(ServerHandshaker.java:1301)
	at sun.security.ssl.ServerHandshaker.chooseCipherSuite(ServerHandshaker.java:1105)
	at sun.security.ssl.ServerHandshaker.clientHello(ServerHandshaker.java:806)
	at sun.security.ssl.ServerHandshaker.processMessage(ServerHandshaker.java:226)
	at sun.security.ssl.Handshaker.processLoop(Handshaker.java:984)
	at sun.security.ssl.Handshaker$1.run(Handshaker.java:924)
	at sun.security.ssl.Handshaker$1.run(Handshaker.java:921)
	at java.security.AccessController.doPrivileged(Native Method)
	at sun.security.ssl.Handshaker$DelegatedTask.run(Handshaker.java:1379)
	at org.glassfish.grizzly.ssl.SSLUtils.executeDelegatedTask(SSLUtils.java:274)
	at org.glassfish.grizzly.ssl.SSLBaseFilter.doHandshakeStep(SSLBaseFilter.java:708)
	at org.glassfish.grizzly.ssl.SSLBaseFilter.doHandshakeStep(SSLBaseFilter.java:622)
	at org.glassfish.grizzly.ssl.SSLBaseFilter.handleRead(SSLBaseFilter.java:334)
	at org.glassfish.grizzly.filterchain.ExecutorResolver$9.execute(ExecutorResolver.java:119)
	at org.glassfish.grizzly.filterchain.DefaultFilterChain.executeFilter(DefaultFilterChain.java:284)
	at org.glassfish.grizzly.filterchain.DefaultFilterChain.executeChainPart(DefaultFilterChain.java:201)
	at org.glassfish.grizzly.filterchain.DefaultFilterChain.execute(DefaultFilterChain.java:133)
	at org.glassfish.grizzly.filterchain.DefaultFilterChain.process(DefaultFilterChain.java:112)
	at org.glassfish.grizzly.ProcessorExecutor.execute(ProcessorExecutor.java:77)
	at org.glassfish.grizzly.nio.transport.TCPNIOTransport.fireIOEvent(TCPNIOTransport.java:539)
	at org.glassfish.grizzly.strategies.AbstractIOStrategy.fireIOEvent(AbstractIOStrategy.java:112)
	at org.glassfish.grizzly.strategies.WorkerThreadIOStrategy.run0(WorkerThreadIOStrategy.java:117)
	at org.glassfish.grizzly.strategies.WorkerThreadIOStrategy.access$100(WorkerThreadIOStrategy.java:56)
	at org.glassfish.grizzly.strategies.WorkerThreadIOStrategy$WorkerThreadRunnable.run(WorkerThreadIOStrategy.java:137)
	at org.glassfish.grizzly.threadpool.AbstractThreadPool$Worker.doWork(AbstractThreadPool.java:593)
	at org.glassfish.grizzly.threadpool.AbstractThreadPool$Worker.run(AbstractThreadPool.java:573)
	at java.lang.Thread.run(Thread.java:748)
]]

```
## Steps to reproduce (Only for bug reports) 

1 -**  Start the domain

    ./asadmin start-domain

2 -**  Open https://localhost:8181 with chrome or firefox

-->

## Environment ##

- **Payara Version**: Payara Server 5.0.0.Alpha3 #badassfish (build 83)
- **Edition**: Full
- **JDK Version**: 8 u144 Oracle
- **Operating System**: Windows / Mac

**Comment by smillidge** [comment]
please remove the -Xbootclasspath setting in the JVM Options in the administration console for the server-config and default-config and then test again.

**Comment by ionutursuleanu** [comment]
Hi,

The removal of -Xbootclasspath fixed the issue. The reason I was using https is that I wanted to test http 2. I tried with java 9 & payaradomain also, but the default version is 1.1. Could you provide some guidance towards enabling http 2? 

Thanks

**Comment by smillidge** [comment]
For advice and guidance please use the Payara Forum https://groups.google.com/forum/#!forum/payara-forum

**Comment by smillidge** [comment]
Fixed by #2148
